src/core/admittance_matrix.jl

```diff
@@ -162,14 +162,14 @@ function calc_susceptance_matrix_inv(data::Dict{String,<:Any})
     S[ref_idx, :] .= 0.0
     S[:, ref_idx] .= 0.0
     S[ref_idx, ref_idx] = 1.0
-    
-    F = LinearAlgebra.ldlt(Symmetric(S); check=false)
+
+    F = LinearAlgebra.ldlt(LinearAlgebra.Symmetric(S); check=false)
     if !LinearAlgebra.issuccess(F)
         Memento.error(_LOGGER, "Failed factorization in calc_susceptance_matrix_inv")
     end
-    M = F \ Matrix(1.0I, num_buses, num_buses)
+    M = F \ Matrix{Float64}(LinearAlgebra.I(num_buses))
```

Comment:
Would like to understand this change.

Reply:
I tried `M = F \ LinearAlgebra.I(num_buses)`, but I got a weird method error.

We could have kept `M = F \ Matrix(1.0 * LinearAlgebra.I, num_buses, num_buses)`, but I prefer the `I(n)` constructor.

I dislike the non-LinearAlgebra prefixed `I` because it gets too easily confused with a bunch of other `I`, particularly when we have `I, J, V` for sparse matrices.